ztrouy/game-rater-api
# Setup - Initial Models and Fixtures

### Purpose
To setup the models and seed data for Game Rater

### Files Changed
- Created file, and wrote initial model and default print output in new `game.py`, `category.py`, `game_category.py`, `review.py`, `rating.py`, and `game_picture.py`
- Imported classes for Game, Category, Game Category, Review, Rating, and Game Picture in `__init__.py` package module in the `/models` directory
- Created initial seed data in new `games.json`, `categories.json`, `game_categories.json`, `reviews.json`, and `ratings.json` fixtures
- Added commands to load seed data for Games, Categories, Game Categories, Reviews, and Ratings in `seed_database.sh`
- Added pillow to dependencies for handling image uploads in `Pipfile` and `Pipfile.lock`

### To Test
Fetch, setup, create or recreate database with `seed_database.sh`, and then confirm the following
- Game table includes 10 Games
- Categories table includes 10 Categories
- Game Categories table includes 10 Game Categories
- Reviews table includes 10 Reviews
- Ratings table includes 25 Ratings

closes #1
